### PR TITLE
Fixes #21266: Cleanup ncf repo

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/21266

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: README.md [hunk 1]
```diff
@@ -33,9 +33,7 @@ There are several layers in this framework, from 10 to 60, where each layer is a
...
-  - __40_it_ops_knowledge__: This directory contains default values for services, like package names for a specific service across different distributions (aka "is it httpd or apache2?"), paths to binaries, default configuration values for services, etc.
-  - __50_techniques__: This directory contains Techniques, which combine generic_methods and it_ops_knowledge to achieve system configurations you need. They may be generic ("Configure OpenSSH server") or specific ("Install and configure our in-house app"). The above example is a "technique".
-  - __60_services__: This directory contains high-level service definitions, which it implements by calling individual techniques, with parameters and conditions. This level is specific for each organisation, and is intended to define services such as "Corporate web site" rather than "Apache config".
+  - __50_techniques__: This directory contains Techniques, which uses generic_methods to achieve system configurations you need. They may be generic ("Configure OpenSSH server") or specific ("Install and configure our in-house app").
```
Comment (ncharles) [inline README.md:36]: we should remove it, nothing ever use it

**Review by ncharles — CHANGES_REQUESTED** [review]
You should drop the mention about 50_techniques in the main README.md

File: tree/10_ncf_internals/initialization.cf [hunk 2]
```diff
@@ -46,12 +46,6 @@ bundle agent initialization
   reports:
     info::
       "${configuration.info} Starting CFEngine ${sys.cf_version} on host ${sys.fqhost} (${sys.flavour} ${sys.arch})";
-    debug::
-      "${configuration.debug} File read from 10_ncf_internals: ${inputs.ncf_internals_files}";
```
Comment (amousset) [inline tree/10_ncf_internals/initialization.cf:50]: these are not used in promises.st anymore

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**